docs/accessibility/core-concepts/run-level-reports.mdx

```diff
@@ -19,75 +19,85 @@ In this document we'll review the three main areas to pay attention to when firs
+The number of snapshots captured for a View. A Snapshot represents the live HTML and CSS state of the application at a specific moment during a recorded test. Multiple snapshots are captured as tests interact with elements, modify state, or load dynamic content.
 
-#### Score
+### Score
 
-An overall score is calculated based on all the Snapshots for a given View. The score for each snapshot is a ratio of passed checks to failed checks, weighted according to the impact of the issues. This produces a top-line percentage score that indicates a general sense of accessibility for a project. For more details, see the [Accessibility Score section](/accessibility/core-concepts/accessibility-score).
+The overall score for a View is a weighted ratio of passed versus failed checks across all snapshots, reflecting the general accessibility level for that segment of the application. This produces a top-line percentage score that indicates a general sense of accessibility for a project. Learn more in the [Accessibility Score section](/accessibility/core-concepts/accessibility-score)..
 
-#### Elements
+### Elements
 
-The count of elements in a view that failed a Rule, had an inconclusive check for a Rule, or were explicitly ignored by configuration.
+The count of elements in a View categorized as:
 
-- **Failed** elements had at least one accessibility Rule fail during the run. These are considered in your accessibility Score and must be fixed.&#x20;
-- **Inconclusive** elements had at least one check marked "incomplete" by Axe Core® during the run. These represent checks that couldn't be completed for technical reasons, or items that need manual review to confirm if there is an accessibility issue. These do not affect your score.
-- **Ignored** elements are those that had either failing or incomplete checks, but were ignored with [CSS selectors in your configuration](/accessibility/configuration/elementfilters). They do not affect your score. Cypress still processes these elements and makes the results available, so that you always know results were was ignored, since these elements are still present in your UI and users will encounter them.
+- **Failed:** Elements violating at least one rule. These affect your accessibility score.
+- **Inconclusive:** elements had at least one check marked "incomplete" by Axe Core® during the run. These represent checks that couldn't be completed for technical reasons, or items that need manual review to confirm if there is an accessibility issue. These do not affect your accessibility score.
+- **Ignored:** Elements with failed or incomplete checks that were excluded via your [configuration](/accessibility/configuration/elementfilters) but still processed. They do not affect your accessibility score.
```

Comment:
I think keeping in the CSS selectors aspect in some way is important here - an important nuance with the is behavior is that it only happens when the elements are ignored. "failed or incomplete checks that were excluded via your [configuration]..." reads more like it's the exclusion of the checks themselves that triggers this. But when a rule is ignored, we really do not run it or collect data about it (at the moment).